src/serialize.js

```diff
@@ -25,8 +24,8 @@ export default function serialize (color, {
 
 	let formatId = format;
 	format = color.space.getFormat(format)
-		   ?? color.space.getFormat("default")
-		   ?? ColorSpace.DEFAULT_FORMAT;
+		    ?? color.space.getFormat("default")
+		    ?? ColorSpace.DEFAULT_FORMAT;
```

Comment:
Same as the && one, neither of this is good formatting. I'd instead right align the `??` with the `=`.